packages/build-tools/src/ios/pod.ts

```diff
@@ -9,28 +8,6 @@ import { BuildContext } from '../context';
 export async function installPods<TJob extends Ios.Job>(ctx: BuildContext<TJob>): Promise<void> {
   const iosDir = path.join(ctx.reactNativeProjectDirectory, 'ios');
 
-  if (ctx.env.EAS_BUILD_COCOAPODS_CACHE_URL) {
```

Comment:
Do not include commits from other reviews, like that. If you want to have stacked PR then this PR should point to the branch where base commit is from, but in this case there is no need for that because changes are not related to each other

Reply:
Sorry, I accidentally did it, it wasn't intentional action